presto-hive/src/main/java/io/prestosql/plugin/hive/InternalHiveSplit.java

```diff
@@ -275,14 +267,10 @@ public long getEnd()
             return addresses;
         }
 
-        public int getEstimatedSizeInBytes()
+        public long getEstimatedSizeInBytes()
         {
-            int result = INSTANCE_SIZE;
-            result += sizeOfObjectArray(addresses.size());
-            for (HostAddress address : addresses) {
-                result += HOST_ADDRESS_INSTANCE_SIZE + address.getHostText().length() * Character.BYTES;
-            }
-            return result;
+            return INSTANCE_SIZE
```

Comment:
We could add a helper `estimatedSizeOf(HostAddress)`
```java
return INSTANCE_SIZE + estimatedSizeOf(addresses, InternalHiveBlock::estimatedSize);

Reply:
Within this class, it doesn't help much. Maybe we could have a new, reusable `SizeOf` for guava classes, but i don't think this is warranted yet.